plugin/src/main/java/org/opennms/velocloud/configuration/VelocloudPollerConfigurationExtension.java

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2022 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2022 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.velocloud.configuration;
+
+import java.util.List;
+
+import org.opennms.integration.api.v1.config.poller.Monitor;
+import org.opennms.integration.api.v1.config.poller.Package;
+import org.opennms.integration.api.v1.config.poller.PollerConfigurationExtension;
+import org.opennms.integration.api.xml.ClasspathPollerConfigurationLoader;
+
+public class VelocloudPollerConfigurationExtension implements PollerConfigurationExtension {
+        final PollerConfigurationExtension pollerConfiguration = new ClasspathPollerConfigurationLoader(VelocloudPollerConfigurationExtension.class, "",
+                "velocloud-poller-configuration.xml").getPollerConfiguration();
```

Comment:
There is already a `poller-configuration.xml` in resources. Should it be renamed and does it have the right syntax/format?

Reply:
in Integration-API in poller-configuration.xsd is specified that `package`has to have one `rrd

Reply:
Which ```rra```s should be specified there?

Reply:
...and at least one ```downtime``` is also needed

Reply:
lets use the default from the horizon poller-configuration for now